app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class Yotpo_Yotpo_Block_Catalog_Product_View extends Mage_Catalog_Block_Product_View {
+
+    const CATEGORY_BOTTOMLINE = 'yotpo/yotpo_general_group/enable_bottomline_category_page';
+
+    public function getReviewsSummaryHtml(Mage_Catalog_Model_Product $product, $templateType = false, $displayIfNoReviews = false) {
+
+        $enableBottomlineCategoryPage = Mage::getStoreConfig(self::CATEGORY_BOTTOMLINE);
```

Comment:
The config here is seems to be the same as in category page (list.php), is it on purpose?  Isn't this view rendering on the product page?

Reply:
enable/disable configuration are same throughout the module. Here we are extending Block of Product for Reviews While at category page we are extending product listing.

Reply:
👍